### PR TITLE
feat: rename pos vel acc

### PR DESCRIPTION
**Pull request by nstarman**
Shorten the names of most classes. Meaning still obvious.
When we add docs (hopefully soon) this will go into the conventions page.

@adrn what do you think?

### CODE REVIEW

File: src/coordinax/_src/base/__init__.py [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
-    "AbstractPosition",
-    # Velocity
-    "AbstractVelocity",
-    # Acceleration
-    "AbstractAcceleration",
+    # Pos
+    "AbstractPos",
+    # Vel
+    "AbstractVel",
+    # Acc
```
Comment (adrn) [inline src/coordinax/_src/base/__init__.py:11]: Total nit pick, but comments should probably stay as full words Position, Velocity, Acceleration.

**Review by adrn — COMMENTED** [review]
I'm not strongly opposed or in favor of this, but want to talk it through. One other place where words could be shortened is in the prefix (Cart/Sph/Cyl/etc.). I'm curious to hear if you have any thoughts about pros/cons to abbreviating the prefix instead of pos/vel/acc, or both, or neither.

**Comment by nstarman** [comment]
I'm not opposed to shortening the prefix either / instead. The goal was to see if it's worth shortening the otherwise long names. I'm still not sure shortening is a good idea 😅.

File: src/coordinax/_src/base/__init__.py (hunk 1 above)
Comment (nstarman) [reply src/coordinax/_src/base/__init__.py:11]: Agreed. Product of global find & replace

**Comment by adrn** [comment]
We discussed this in person and decided to move forward with the name shortening.

**Pull request merged**